playground/doc/playground.mdx

```diff
@@ -19,19 +47,19 @@ import { Button } from './Button';
 <Playground
   scope={{ Button }}
   code={`
-  <Button>Button</Button>
-  <Button primary>Primary</Button>
-  <Button danger>Danger</Button>
+<Button>Button</Button>
+<Button primary>Primary</Button>
+<Button danger>Danger</Button>
   `}
 />;
 ```
 
 <Playground
   scope={{ Button }}
   code={`
-<Button>Button</Button> 
-<Button primary>Primary</Button> 
-<Button danger>Danger</Button> 
```

Comment:
my favourite part about this PR: find 10 differences between how it should really look like (1st) and how it looks when global styles leak unexpectedly (2nd) (please ignore code differences, the CSS applied to both demos buttons is exactly the same)

![image](https://user-images.githubusercontent.com/137844/127203419-d449fb3f-d0c7-4e73-bbcb-4a16a1bdb5ed.png)